propagators/opentelemetry-propagator-opentracing/src/OpenTracingPropagator.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Baggage,
+  Context,
+  getBaggage,
+  getSpanContext,
+  isSpanContextValid,
+  isValidSpanId,
+  isValidTraceId,
+  setBaggage,
+  setSpanContext,
+  TextMapGetter,
+  TextMapPropagator,
+  TextMapSetter,
+  TraceFlags,
+} from '@opentelemetry/api';
+
+/** OT header keys */
+export const OT_TRACE_ID_HEADER = 'ot-tracer-traceid';
+export const OT_SPAN_ID_HEADER = 'ot-tracer-spanid';
+export const OT_SAMPLED_HEADER = 'ot-tracer-sampled';
+export const OT_BAGGAGE_PREFIX = 'ot-tracer-baggage-';
```

Comment:
Is there somewhere a specification for baggage in opentracing? I haven't found that much about this but at some places prefix `ot-baggage-` is used instead.

Reply:
`ot-baggage-` is correct, thanks!